web/frontend/src/components/slam/index.svelte

```diff
@@ -278,13 +277,18 @@ const toggleAxes = () => {
 
 const handleMoveClick = async () => {
   try {
-    const base = bases[0]!;
-    await moveOnMap(
-      $robotClient,
+    // set pose in frame
+    const lastPose = await slamClient.getPosition();
+    const destinationMM = lastPose.pose!;
+    destinationMM.x = destination!.x * 1000;
+    destinationMM.y = destination!.y * 1000;
+
+    await motionClient.moveOnMap(
+      destinationMM,
+      bases[0]!,
       slamResourceName,
-      base,
-      destination!.x,
-      destination!.y
+      { planDeviationM: 0.5 },
```

Comment:
This is failing a lint error despite working in the browser:
https://github.com/viamrobotics/rdk/actions/runs/7873963113/job/21482471651?pr=3574#step:5:2176

Also, the typescript SDK sets this value in the MotionConfiguration type: https://github.com/viamrobotics/viam-typescript-sdk/blob/main/src/services/motion/types.ts#L88

Do I need to do something in the typescript SDK to placate the type checker?

@micheal-parks @ethanlook

Reply:
From a quick glance it appears that those addtional props (`obstacleDetectorsList`, `positionPollingFrequencyHz`) are not marked as optional and should be? Something like:

```ts
interface MoveOnMapOptions {
  planDeviationM: number // if required
  obstacleDetectorsList?: number
  positionPollingFrequencyHz?: number
  ...
}

Reply:
1. I don't see the text string `MoveOnMapOptions` in the typescript SDK nor in the RC web app
2. The type in question is 
```
import pb from '../../gen/service/motion/v1/motion_pb';
...
export type MotionConfiguration = pb.MotionConfiguration.AsObject;
``` 
https://github.com/viamrobotics/viam-typescript-sdk/blob/main/src/services/motion/types.ts#L9

That object (at least in vanilla js) knows about `planDeviationM`: https://github.com/viamrobotics/api/blob/main/gen/js/service/motion/v1/motion_pb.js#L1956 How do I get typescript to believe that that field is a member of the `MotionConfiguration` type?